src/groundlight/client.py

```diff
@@ -225,10 +242,27 @@ def submit_image_query(  # noqa: PLR0913 # pylint: disable=too-many-arguments
             iq_id = self.api_client.submit_image_query_with_inspection(**params)
             image_query = self.get_image_query(iq_id)
 
-        if wait:
+        if wait > 0:
             threshold = self.get_detector(detector).confidence_threshold
             image_query = self.wait_for_confident_result(image_query, confidence_threshold=threshold, timeout_sec=wait)
-        return self._fixup_image_query(image_query)
+
+        return self._fixup_image_query(image_query, want_async=want_async)
+
+    def ask_async(
+        self,
+        detector: Union[Detector, str],
+        image: Union[str, bytes, Image.Image, BytesIO, BufferedReader, np.ndarray],
+        human_review: Optional[str] = None,
+        inspection_id: Optional[str] = None,
+    ) -> ImageQuery:
+        """
+        Convenience method for submitting an image query asynchronously. This is equivalent to calling
+        submit_image_query with want_async=True and wait=0. Use get_image_query to retrieve the result of the image
+        query.
+        """
```

Comment:
We should flesh this out with full documentation of all the parameters accepted.

I have a bunch of thoughts of how to word the summary here, but I’d rather type them when I’m at a proper keyboard.

Reply:
I copy pasted the relevant descriptions from submit_image_query for now.